### PR TITLE
fix: [#187636181] synchronize migration and userData for interstate moving and logistics

### PR DESCRIPTION
**Pull request by SirSamuelJoseph**
## Description

UserData and the versioned UserData in the migrations folder got misaligned in this commit: https://github.com/newjersey/navigator.business.nj.gov/commit/4f574aaf6314b94b9a67246a475ece020589d66e

Basically, `interstateLogistics` and `interstateMoving` are the profileData fields that are being referenced in the application, but they are called `isInterstateLogisticsApplicable` and `isInterstateMovingApplicable` in the migrations version of user data. This restores the two to the same property name.

As for populating data, these fields were only assigned on industry assignment. Therefore, the migration can just apply the same industry check to set the correct values for these two fields

### Ticket

[Replace with ticket_id](https://www.pivotaltracker.com/story/show/)

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see [CMS Additions in Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po/edit#heading=h.fu5jdsrcqxbh))
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: api/src/db/migrations/v134_added_planned_renovation_question_field.ts [hunk 1]
```diff
@@ -484,3 +485,260 @@ type v134GetFilingResponse = {
   standingDoc: string;
   certifiedDoc: string;
 };
+
+// ---------------- v134 factories ----------------
```
Comment (SirSamuelJoseph) [inline api/src/db/migrations/v134_added_planned_renovation_question_field.ts:489]: Just added factories to the last migration file

File: api/src/db/migrations/v135_use_interstate_logistics_and_transport_fields.ts [hunk 2]
```diff
@@ -0,0 +1,761 @@
...
+      Object.values(v134Data.businesses)
+        .map((business: v134Business) => migrate_v134Business_to_v135Business(business))
+        .map((currBusiness: v135Business) => [currBusiness.id, currBusiness])
+    ),
+    version: 135,
+  } as v135UserData;
+};
+
+const migrate_v134Business_to_v135Business = (business: v134Business): v135Business => {
+  const getLogisticsResponse = (): boolean => {
```
Comment (SirSamuelJoseph) [inline api/src/db/migrations/v135_use_interstate_logistics_and_transport_fields.ts:38]: This and `getMovingResponse` use the same logic as the initial migration (v107)

File: api/src/db/migrations/v135_use_interstate_logistics_and_transport_fields.ts [hunk 3]
```diff
@@ -0,0 +1,761 @@
...
+  homeBasedBusiness: boolean | undefined;
+  providesStaffingService: boolean;
+  certifiedInteriorDesigner: boolean;
+  realEstateAppraisalManagement: boolean;
+  cannabisLicenseType: v135CannabisLicenseType;
+  cannabisMicrobusiness: boolean | undefined;
+  constructionRenovationPlan: boolean | undefined;
+  carService: v135CarServiceType | undefined;
+  interstateTransport: boolean;
+  interstateLogistics: boolean | undefined;
```
Comment (SirSamuelJoseph) [inline api/src/db/migrations/v135_use_interstate_logistics_and_transport_fields.ts:16]: `interstateLogistics` and `interstateMoving` are the new fields

**Review by somabadri — APPROVED** [review]
looks good to me!

**Pull request merged**